services/api/src/lib/utils/schema.js

```diff
@@ -57,3 +63,46 @@ function isDisallowedField(doc, key, allowPrivate = false) {
   }
   return false;
 }
+
+exports.loadModel = (definition, name) => {
```

Comment:
Kinda seems like loading models isn't the job of this simple `util` for building schemas.

What if instead of exporting `loadModels` and forcing the importer to do it, we just have a single file `src/models/index.js` that basically does all of this immediately? ie. it loads all JSON schemas, builds the mongoose models and exports them?

That way we can do this directly:

```js
const { User } = require('./src/models');
```

basic CJS `require` behavior will prevent them from being loaded again if required elsewhere and we can hide all the logic converting JSON to Mongoose.

I guess for testing we could also export `loadModel` just to test the functionality as well

---------

One other kind of related note is I was creating a cronjob the other day and having issues because importing only one model doesn't import the other ones it depends on (via `ObjectId`) so loading all models at once is nice to prevent that no matter how we do it